python/paddle/fluid/framework.py

```diff
@@ -603,7 +603,7 @@ def convert_np_dtype_to_dtype_(np_dtype):
     elif dtype == np.bool:
         return core.VarDesc.VarType.BOOL
     elif dtype == np.uint16:
-        return core.VarDesc.VarType.INT16
+        return core.VarDesc.VarType.BF16
```

Comment:
Why change here?

Reply:
There is no support for bfloat16 in NumPy.  
We would like to test bfloat16 operators not only with input data float32, but also with bfloat16, so it converts `np.uint16` data to C++ `bfloat16`.  
It was also added to `tensor_py.h` https://github.com/PaddlePaddle/Paddle/blob/c0caf0e45fac2bbbae184a702a4fe0313a9c3c69/paddle/fluid/pybind/tensor_py.h#L272 
Importantly, there is no other usage of `np.uint16` in Paddle.